en/simulation/gazebo.md

```diff
@@ -258,19 +260,58 @@ The next time you build/restart Gazebo it will use the new GPS noise setting.
 
 By default Gazebo displays a flat featureless plane, as defined in [empty.world](https://github.com/PX4/sitl_gazebo/blob/master/worlds/empty.world).
 
-You can load any of the worlds in [PX4/sitl_gazebo/worlds](https://github.com/PX4/sitl_gazebo/tree/master/worlds)) by specifying them as the final option in the PX4 configuration target.
+You can load any of the worlds in [PX4/sitl_gazebo/worlds](https://github.com/PX4/sitl_gazebo/tree/master/worlds) by specifying them as the final option in the PX4 configuration target:
+```
+make px4_sitl gazebo_<model_name>__<world_name>
+```
+
 For example, to load the *warehouse* world, you can append it as shown:
 ```
 make px4_sitl_default gazebo_plane_cam__warehouse
 ```
 
-> **Note** There are two underscores after the model (`plane_cam`) indicating that the default debugger is used (none).
+> **Note** There are *two underscores* after the model (`plane_cam`) indicating that the default debugger is used (none).
    See [Building the Code > PX4 Make Build Targets](../setup/building_px4.md#make_targets).
 
-
 You can also specify the full path to a world to load using the `PX4_SITL_WORLD` environment variable.
 This is useful if testing a new world that is not yet included with PX4.
 
+> **Tip** If the loaded world does not align with the map, you may need to [set the world location](#set_world_location). 
+  
+
+## Set World Location {#set_world_location}
+
+By default, a world is loaded at the current vehicle location.
```

Comment:
@Jaeyoung-Lim Is this correct?

My assumption is that what happens is that setting the world causes the world to be loaded in the right place AND sets the vehicle to load at that location. 
So if you set the world AND a different custom takeoff location the world/map are still correctly aligned - just the vehicle may not be near them
(the other possibility is that if you set the different location the vehicle position is set there, and the world loads at that location - which would make less sense).

Reply:
The vehicle always get spawned at the origin and the world just defines where the origin lies on gps coordinates.

Reply:
@Jaeyoung-Lim I think that you're saying the vehicle always gets spawned at the origin of the world, and by setting the GPS co-ordinates in the world you're making that origin be at a certain GPS co-ordinate - right?

And by setting the position "manually" you're doing the same thing? I.e. you're not saying "the vehicle is at GPS x, y, z. You're saying "set the origin of the world to position X, Y, Z, and the vehicle will appear there? Is that correct?




so what happens when vehicle position is explicitly set? I THINK you're saying that you're